contributing.adoc

```diff
@@ -24,11 +24,13 @@ the running GUI without restarting. Saving a file with errors may result in the
 running GUI crashing, but that's not a major issue as the GUI is out-of-process
 compared to the synthesis backend.
 
-To enable code hotloading, simply use the `linux-hotload` target instead of the
+To enable code hotloading, simply use the `linux-hotload` for Linux os and  `windows-hotload` for windows os target instead of the
 default one: 
 
 --------------------------------------------------------------------------------
-make linux-hotload
+make linux-hotload for linux
+
+make windows-hotload for windows
```

Comment:
This section is meant to contain only code, which means that most people will expect being able to directly copy-paste the content and use it. What you added at the end of the lines (e.g. "for linux") makes that impossible.

You have at least two options here:
- make things valid by adding a proper comment; `make linux-hotload  # for linux`.
- split the block into two. Something like

```
For Linux:
--------------------------------------------------------------------------------
make linux-hotload
--------------------------------------------------------------------------------

For Windows:
--------------------------------------------------------------------------------
make windows-hotload
--------------------------------------------------------------------------------
```